greenheart/simulation/technologies/steel/steel.py

```diff
@@ -759,8 +849,108 @@ def run_steel_finance_model(
     summary = pf.get_summary_vals()
     price_breakdown = pf.get_cost_breakdown()
 
+    if config.save_plots or config.show_plots:
```

Comment:
The output processing/manipulation/plotting should be moved elsewhere, in my opinion. That falls more into project code than simulation code. Another note to discuss in the future.